code/game/atoms.dm

```diff
@@ -431,8 +433,9 @@ var/list/blood_splatter_icons = list()
 //effects at world start up without causing runtimes
 /atom/proc/spawn_atom_to_world()
 
-//This will be called after the map and objects are loaded
-/atom/proc/initialize()
+//Called after New if the world is not loaded with TRUE
+//Called from base of New if the world is loaded with FALSE
+/atom/proc/Initialize(roundstart)

```

Comment:
World init speed up idea:

Make this a set waitfor = 0 proc so sleeps in here don't slow down initializing anything else.